app/views/users/edit.html.erb

```diff
@@ -0,0 +1,58 @@
+<!--
+<form_tag accept-charset="UTF-8" action="/users_path" method="post">
+    <input type="hidden" name="authenticity_token" value="<%= form_authenticity_token %>" />
+    <label for="username">Username:</label>
+    <input type="text" id="username" name="user[username]" /><br />
+    <br />
+    <label for="email">Email:</label>
+    <input type="email" id="email" name="user[email]" /><br />
+    <br />
+    <label for="email">Password:</label>
+    <input type="password" id="password" name="user[password]" /><br />
+    <br />
+    <input type="submit" value="Submit" />
+</form>
+
+<%= form_tag("/users_path", method: "post") do %>
+    <%= label_tag(:username, "Username:") %>
+    <%= text_field_tag(:username) %><br>
+
+    <%= label_tag(:email, "email:") %>
+    <%= text_field_tag(:email) %><br>
+
+    <%= label_tag(:password, "password:") %>
+    <%= password_field_tag(:password) %><br>
+
+    <%= submit_tag("Submit") %>
+<% end %>
+-->
+<%= form_for @user do|f|%>
+    <input type="hidden" name="form_authenticity_token" value="<%= form_authenticity_token %>" />
+    <% if @user.errors.any? %>
+        <div id="error_explanation">
+            <h2><%= pluralize(@user.errors.count, "error") %> 
+            prohibited this user from being edited</h2>
+
+            <ul>
+                <% @user.errors.full_messages.each do |msg| %>
+                    <li><%= msg %></li>
+                <% end %>
+            </ul>
+        </div>
+    <% end %>
```

Comment:
Let's remove this as it is not requirement, but you can implement this in future projects

Reply:
Done!